### PR TITLE
Fix button styling in ag grid filter panel

### PR DESCRIPTION
**Pull request by Fercas123**
add `ag-standard-button` styles to salt-ag-theme

path to see changes: 
https://31a01c59.saltdesignsystem-storybook.pages.dev/?path=/story/data-grid-ag-grid-theme--basic-grid

### CODE REVIEW

**Comment by origami-z** [comment]
@pedroBlazquez can you help whether the buttons are aligned with what you have right now?

**Review by joshwooding — CHANGES_REQUESTED** [review]
Can you add a Changeset please

**Review by joshwooding — APPROVED** [review]

File: .changeset/short-shrimps-reflect.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/ag-grid-theme": minor
+---
+
+Add salt styles to `.ag-standard-button`.
```
Comment (joshwooding) [inline .changeset/short-shrimps-reflect.md:5]: Can you update the changeset to describe the issue that is being fixed please

**Review by joshwooding — CHANGES_REQUESTED** [review]

**Review by joshwooding — DISMISSED** [review]

File: packages/ag-grid-theme/css/_salt-ag-theme-mixin.scss [hunk 2]
```diff
@@ -333,6 +333,28 @@
     width: var(--salt-size-stackable);
   }
 
+  .ag-standard-button {
```
Comment (ZarkoPernar) [inline packages/ag-grid-theme/css/_salt-ag-theme-mixin.scss:336]: I assume it doesn't have a focus ring?

Comment (Fercas123) [reply packages/ag-grid-theme/css/_salt-ag-theme-mixin.scss:336]: nice catch, I'll add that in

**Review by ZarkoPernar — APPROVED** [review]

**Review by origami-z — APPROVED** [review]
Difference between UITK now is whether they're centered. Will leave to future alignment.

**Pull request merged**